include/boost/wave/util/cpp_macromap.hpp

```diff
@@ -1004,13 +1023,56 @@ bool adjacent_stringize = false;
                     impl::replace_ellipsis(expanded_args, i, expanded, pos);
                 }
                 else
+
+#if BOOST_WAVE_SUPPORT_VA_OPT != 0
+                if (is_va_opt) {
```

Comment:
I using `__VA_OPT__` without ellipses in the macro definition parameter list valid?

Reply:
P0306r4 states that `__VA_OPT__` may only be used in the replacement text of a variadic macro.

Reply:
No, and I have [a test](https://github.com/jefftrull/wave/blob/9121d11e20adb392160f2d4ec8f19a64a901677b/test/testwave/testfiles/t_8_007.cpp#L15) for that.